tests/selectors/onboarding.selectors.ts

```diff
@@ -2,7 +2,7 @@ export enum OnboardingSelectors {
   AllowAnalyticsBtn = 'allow-analytics-btn',
   BackUpSecretKeyBtn = 'back-up-secret-key-btn',
   DenyAnalyticsBtn = 'deny-analytics-btn',
-  HiroWalletLogoRouteToHome = 'hiro-wallet-logo-route-to-home',
+  LeatherLogoRouteToHome = 'hiro-wallet-logo-route-to-home',
```

Comment:
Should we update `hiro-wallet-logo-route-to-home` here too?

Reply:
O yeah, good catch. Will update.